src/main/java/useCases/Calculator/HistogramPanel.java

```diff
@@ -0,0 +1,168 @@
+package useCases.Calculator;
```

Comment:
The formatting is different! The opening curly brace appears on a different line which (I think) is C/C++ formatting.

Reply:
Extending on this, this class seems to be imported then modified(?)